CONFIG.md

```diff
@@ -147,18 +147,28 @@ ios:
 
   # [optional] Parameters for exporting typography
   typography:
-    # [optional] Absolute or relative path to swift file where to export UIKit fonts (UIFont extension).
+    # Choose what font system you want to use SwiftUI or UIKit
+    fontSystem: UIKit
```

Comment:
How do you allow both at the same time?

Reply:
Actually I was wondering if we could just remove the `fontSystem` param (See my question 1 above), wondering your thoughts on this ?